### ISSUE TITLE
Game setup

### ISSUE THREAD

**Issue by jnawaz** [jnawaz/squashmarker#1]
- [x] Allow name input for two players (Player A, Player B)
- [x] Game setup details according to the spec (see README for more details)
- [ ] ~~Select who starts with the serve (should allow for switching before starting the game in case of selecting wrong player to serve) -~~ postponed this to the game scoring screen.
- [x] Start the game
